ui/components/ConnectionWizard/ConfigComponents/Kubernetes.js

```diff
@@ -29,14 +29,12 @@ const KubernetesConfig = ({
 
   const handleChange = (name) => {
     return (event) => {
+      let fileInput = document.querySelector("#k8sfile");
+      let k8sfile = fileInput.files[0]
```

Comment:
did you verified that it is working?

Reply:
@Abhishek-kumar09  The successful snackbar comes but it doesn't show up in the list (even without these changes) , would you test it.